### ISSUE TITLE
Some Questions of Your Paper

### ISSUE THREAD

**Issue by mcairlangga2** [JosephKJ/OWOD#113]
Dear Joseph,
Thank you for the interesting work you made on your CVPR 2021 paper. I'm new in this open world object detection field. I have a confusion about how ORE works and want to ask some questions.

1. What is actually the main purpose of the clustering process? I understand how the clustering works. However, after the feature successfully clustered, what is the next step? Is there anything related with the classification and regression head at the end? Does the clustering also take part in the evaluation (testing) step? 

Also, when the model is already trained on, let's say, Task 1, does the cluster formed in this task be kept for the next task and the clustering process continues until there are 80 clusters or is it done separately for each task?

2. Why do you estimate the energy distribution for known and unknown using a validation set? Why not use the training one?

3. When is the exemplar replay based fine-tuning done? Is it on the directly before the evaluation (testing) step after the model trained on the new task? or is it trained together with the new task?

4. How do you get the bounding box for the unknown one? Is it directly from the unknown-aware RPN?

Hope you can help me to understand this.
Thank you.
